applications/FemToDemApplication/custom_utilities/aitken_relaxation_utility.hpp

```diff
@@ -0,0 +1,489 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics FemDem Application
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//                   Ruben Zorrilla
+//
+
+#if !defined(KRATOS_AITKEN_RELAXATION_UTILITY)
+#define  KRATOS_AITKEN_RELAXATION_UTILITY
+
+// System includes
+
+// External includes
+#include "utilities/math_utils.h"
+#include "spaces/ublas_space.h"
+#include "fem_to_dem_application_variables.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+///@}
+
+///@name Type Definitions
+///@{
+///@}
+
+///@name  Enum's
+///@{
+///@}
+
+///@name  Functions
+///@{
+///@}
+
+///@name Kratos Classes
+///@{
+
+/** @brief Aitken relaxation technique for FSI PFEM-FEM-DEM coupling
+ */
+class AitkenRelaxationUtility
```

Comment:
this exists both in the FSIApplication and the CoSimulationApplication, why not using those?

Reply:
What's the difference?, or can you derive from it in case of different?